packages/compute-baseline/src/browser-compat-data/release.test.ts

```diff
@@ -87,4 +97,37 @@ describe("Release", function () {
       assert.equal(safariPreview.isPrerelease(), true);
     });
   });
+
+  describe("inRange()", function () {
+    it("throws when comparing between two browsers", function () {
+      const cr = browser("chrome");
+      const fx = browser("firefox");
+
+      assert.throws(() => cr.version("50").inRange(fx.version("50")), Error);
+    });
+
+    it("handles closed ranges", function () {
```

Comment:
Can you add a test for when the version equals the end version, to show the range is exclusive of the end point?

Reply:
Done with 7c2bbadf46f97d8ab98e20bc2b23202461cde0cf.